python/metatensor-core/metatensor/core/block.py

```diff
@@ -289,20 +303,23 @@ def add_gradient(self, parameter: str, gradient: "TensorBlock"):
         Add gradient with respect to ``parameter`` in this block.
 
         :param parameter:
-            add gradients with respect to this ``parameter`` (e.g.
-            ``positions``, ``cell``, ...)
+            add gradients with respect to this ``parameter`` (e.g. ``positions``,
+            ``cell``, ...)
 
         :param gradient:
-            a :py:class:`TensorBlock` whose values contain the gradients with
-            respect to the ``parameter``. The labels of the gradient
-            :py:class:`TensorBlock` should be organized as follows: its
-            ``samples`` must contain ``"sample"`` as the first label, with
-            values containing the index of the corresponding ``samples`` in the
-            :py:class:`TensorBlock` containing values; its components must
-            contain at least the same components as the :py:class:`TensorBlock`
-            containing values, with any additional components coming before
-            those; its properties must match exactly those of the
-            :py:class:`TensorBlock` containing values.
+            a :py:class:`TensorBlock` whose values contain the gradients with respect to
+            the ``parameter``. The labels of the gradient :py:class:`TensorBlock` should
+            be organized as follows:
```

Comment:
I am trying to be more clear which tensor block we are referring to (block or gradient block) and which values (block or labels) we are referring to , by using here `this :py:class:TensorBlock values` and in the rest of the text. I am not super happy, but I hope it is an improvement

```suggestion
            a :py:class:`TensorBlock` whose values contain the gradients of this
            :py:class:`TensorBlock` values with respect to the ``parameter``. The labels
            of the gradient :py:class:`TensorBlock` should be organized as follows:
```